src/firebase-namespace.ts

```diff
@@ -335,6 +294,14 @@ export class FirebaseNamespace {
     return Object.assign(fn, { Auth: auth });
   }
 
+  get credential(): FirebaseServiceNamespace<CredentialService> {
```

Comment:
Let's remove this and instead map `credential` to an object/namespace that has top-level functions.